jib-build-plan/src/main/java/com/google/cloud/tools/jib/api/buildplan/FileEntriesLayer.java

```diff
@@ -133,7 +132,7 @@ public Builder addEntry(
       return addEntry(
           sourceFile,
```

Comment:
`addEntry(Path, AbsoluteUnixPath, Instant)` is currently not used. Should it still remain in the class?

Reply:
I'd say yes. `c.g.c.t.j.api.buildplan` is the public API of the "Container Build Plan" library. We may not use the method internally, but external users may be using it (and they always do).

Reply:
Oh I see! Thanks for the explanation:)